revue-dhistoire-des-sciences.csl

```diff
@@ -0,0 +1,311 @@
+    <group delimiter=" ">
+      <text term="op-cit" form="short" font-style="italic"/>
+      <group delimiter=" ">
+        <label variable="first-reference-note-number" form="short"/>
+        <text variable="first-reference-note-number" form="short" font-style="normal"/>
+      </group>
+    </group>
+  </macro>
+  <macro name="publisher">
+    <choose>
+      <if type="chapter book thesis entry-encyclopedia manuscript" match="any">
+        <group delimiter=", " prefix="(" suffix=")">
+          <group delimiter="&#160;: ">
+            <choose>
+              <if type="thesis" match="none">
+                <text variable="publisher-place"/>
+              </if>
+            </choose>
+            <text variable="publisher"/>
+          </group>
+          <text macro="date"/>
+        </group>
+      </if>
+      <else-if type="book" match="any">
+        <group delimiter=", " prefix="(" suffix=")">
+          <text variable="publisher"/>
+          <text macro="date"/>
+        </group>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="date">
+    <date variable="issued">
+      <date-part name="year"/>
+    </date>
+  </macro>
+  <macro name="volume">
+    <group delimiter=" ">
+      <label variable="volume" form="short"/>
+      <number variable="volume" form="roman"/>
+    </group>
+  </macro>
+  <macro name="pageno">
+    <choose>
+      <if type="chapter paper-conference" match="any">
+        <group delimiter=" ">
+          <text variable="page"/>
+        </group>
+      </if>
+      <else>
+        <text variable="page"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="pageref">
+    <group delimiter="&#160;">
+      <label variable="locator" form="short"/>
+      <text variable="locator"/>
+    </group>
+  </macro>
+  <macro name="volref">
+    <group delimiter=" ">
+      <group delimiter="/">
+        <text variable="volume"/>
+        <text variable="issue"/>
+      </group>
+      <text macro="date" prefix="(" suffix=")"/>
+    </group>
+  </macro>
+  <macro name="container">
+    <choose>
+      <if type="chapter paper-conference entry-encyclopedia" match="any">
+        <group delimiter=", ">
+          <group delimiter=" ">
+            <text term="in" font-style="normal"/>
+            <text macro="editor"/>
+          </group>
+          <text variable="container-title" font-style="italic"/>
+        </group>
+      </if>
+      <else>
+        <text variable="container-title" font-style="italic"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="editor">
+    <names variable="editor translator">
+      <name and="text" delimiter-precedes-last="never" initialize="false" initialize-with=". "/>
+      <label form="short" prefix=" (" suffix=")"/>
+    </names>
+  </macro>
+  <macro name="volumes">
+    <group delimiter=" ">
+      <number variable="number-of-volumes"/>
+      <label variable="number-of-volumes" form="short"/>
+    </group>
+  </macro>
+  <macro name="edition">
```

Comment:
replace with this:
`  <macro name="edition">
    <choose>
      <if is-numeric="edition">
        <group delimiter=" ">
          <number variable="edition" form="ordinal"/>
          <label variable="edition" form="short"/>
        </group>
      </if>
      <else>
        <text variable="edition"/>
      </else>
    </choose>
  </macro>`